File: Dev/Frontend/LambdaPP_WebApp_Alpha_3.0/WildTask/src/app/logic/models/Stock.ts
import Low_Stock from "./Low_Stock";
import TaskStockBridge from "./TaskStockBridge";

export default class Stock {

    stock_ID!: number
    stock_Name!: string
    stock_Description!: string
    stock_Quantity!: number;
    stock_Image!: string;
    stock_Enabled!: boolean;
    stock_Lead_Time!: number;
    task_Stock_Bridges!: TaskStockBridge[];
    Low_Stock: Low_Stock|undefined;

    constructor(name: string, desc: string, quantity: number)
    {
        this.stock_Name = name
        this.stock_Description = desc
        this.stock_Quantity = quantity
        this.stock_Enabled = true;
    }
}

File: Mobile/D2/LambdaPP_AndroidApp_D2/app/src/main/java/com/example/lambdapp_androidapp_d2/models/Stock.java
package com.example.lambdapp_androidapp_d2.models;

import java.util.List;

public class Stock {
    public int stock_ID ;
    public String stock_Name;
    public String stock_Description;
    public int stock_Quantity;
    public int stock_Lead_Time;
    public boolean stock_Enabled;
    public List<Task_Stock_Bridge> task_Stock_Bridges;
}
